### DISCUSSION TITLE
Enhancement: 2 frames for parents-christmas icon instead of 1

### DISCUSSION THREAD

**Discussion by DuckBergen** [FunkinCrew/Funkin#1619] [Ideas]
## What is your suggestion, and why should it be implemented?
for people who want to add a frame when parents-christmas's health gets low, it isnt even hard
from
```haxe
animation.add('parents-christmas', [17], 0, false, isPlayer);
```
to
```haxe
animation.add('parents-christmas', [17, 18], 0, false, isPlayer);
```
in HealthIcon.hx

**Comment by treacherousfiend** [comment]
Already fixed (but not yet merged) in #614
